src/earthen/clj_cb/bucket.clj

```diff
@@ -56,6 +56,19 @@
             .toString)
          (document->map doc))))))
 
+(defn lookup-in
+  "Retrieves sub-document values"
+  [bucket id & rest]
+  (let [builder (.lookupIn bucket id)
+        _ (.get builder (into-array rest))
+        result (.execute builder)]
+    (apply merge-with into {} (map-indexed
+                               (fn [idx itm]
```

Comment:
What is idx and itm? index and item? don't be afraid of being more expressive, github don't charge extra for characters :)

Reply:
Sure, will make the change, was just copying from an example using map-indexed.